### PR TITLE
Fix: Commits as per comment

### PR DESCRIPTION
**Pull request by harsshjain**
1) Added GJ and AP dashboard in website_checker.py
2) Tried new format for KL and PY
3) Merged old PR
4) Increased timeout to 18 min

### CODE REVIEW

File: twitter_notifications.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def stream_tweets(api, results, bot, chat_id_data_ops, chat_id_aux, minutes=10):
 
                     message = (
                         f"@{status._json['user']['screen_name']} tweeted :\n\n{url}"
-                        f"\n\n#{result['statename'].replace(' ','')}"
+                        f"\n\n#{result['statename'].replace(' ',' ')}"
```
Comment (sudevschiz) [inline twitter_notifications.py:110]: This would replace space with space. Wouldn't be helpful

File: website_checker.py [hunk 2]
```diff
@@ -68,7 +68,10 @@ def check_now(a,urls):
 	Code to check if a URL has updated
 	'''
 	# For KL
-	today = datetime.now(pytz.timezone('Asia/Calcutta')).strftime('%d/%m/%Y')
+	today_1 = datetime.now(pytz.timezone('Asia/Calcutta')).strftime('%DD/%MM/%YYYY')
```
Comment (sudevschiz) [inline website_checker.py:71]: I dont think this is valid format for strftime.` %d/%m/%Y` would give the date as 05/08/2020

File: website_checker.py [hunk 3]
```diff
@@ -116,6 +119,16 @@ def check_now(a,urls):
 			"name" : "WestBengal",
 			"url" : "https://www.wbhealth.gov.in/pages/corona/bulletin",
 			"check" : 'href="https://www.wbhealth.gov.in/uploaded_files/corona/'
+		},
```
Comment (sudevschiz) [inline website_checker.py:122]: No `check` conditions for GJ and AP? We should add the corresponding frame that would change

**Comment by harsshjain** [comment]
Fixed in New PR except KL and PY Dashboard

**Pull request closed without merging**